src/Metal/MTLEnums.cs

```diff
@@ -598,15 +631,31 @@ public enum MTLFeatureSet : nuint {
 		iOS_GPUFamily2_v3 = 6,
 		[iOS (10,0), NoTV, NoWatch, NoMac]
 		iOS_GPUFamily3_v2 = 7,
+		[iOS (11,0), NoTV, NoWatch, NoMac]
+		iOS_GPUFamily1_v4 = 8,
+		[iOS (11,0), NoTV, NoWatch, NoMac]
+		iOS_GPUFamily2_v4 = 9,
+		[iOS (11,0), NoTV, NoWatch, NoMac]
+		iOS_GPUFamily3_v3 = 10,
 
-		[Mac (10,11)]
-		OSX_GPUFamily1_v1 = 10000,
+		[Mac (10,13), NoiOS, NoTV, NoWatch]
+		macOS_GPUFamily1_v1 = 10000,
 		
-		[NoiOS, NoTV, NoWatch, Mac (10,12)]
-		OSX_GPUFamily1_v2 = 10001,
+		[Availability (Introduced = Platform.Mac_10_11, Deprecated = Platform.Mac_10_13)]
```

Comment:
Add `message: "Use 'macOS_GPUFamily1_v1' instead."` to let the IDE guide the developer.